README.md

```diff
@@ -1 +1,61 @@
-# Budget-App
\ No newline at end of file
+## BUDGET APP
+>Is a mobile web application where you can manage your budget: you have a list of transactions associated with a category, so that you can see how much money you spent and on what
```

Comment:
- [ ] [REQUIRED] I see you have attached the video presentation on the PR description as project documentation. Let's also add the link to the README as it is the file giving the entire documentation of the project. 
Also, one of the project requirements states that you should `highlight a piece of the code` that you are proud of making. Kindly make another video to correct this.